elastalert/alerts.py

```diff
@@ -687,6 +690,12 @@ def alert(self, matches):
         if self.bump_tickets:
             ticket = self.find_existing_ticket(matches)
             if ticket:
+                inactivity_datetime = arrow.now().replace(days=-self.bump_after_inactivity)
```

Comment:
I assume this ticket.fields.updated is a ISO8601 string?
To keep consistent with the rest of the code, can you use util.ts_now and util.ts_to_dt
```
from util import ts_now, ts_to_dt
...
inactivity_datetime = ts_now() - datetime.timedleta(days=self.bump_after_inactivity)
if ts_to_dt(ticket.fields.updated) ...
```